stories/submission.stories.js

```diff
@@ -0,0 +1,118 @@
+// https://github.com/EnCiv/undebate-ssp/issues/51
+import React from 'react'
+import Submission from '../app/components/submission'
+
+export default {
+    title: 'Submission',
+    component: Submission,
+    argTypes: {},
+}
+
+const Template = (args, context) => {
```

Comment:
Please checkout [script.stories.js](https://github.com/EnCiv/undebate-ssp/blob/main/stories/script.stories.js) for how to pass electionOM to submission, rather than electionObj.

The story also shows how to initialize electionObj with defaultElectionObj.